front-end/src/Register/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from "react";
+import { Button, Col, Container, Row, Form } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+
+const Register = () => {
+    const navigate = useNavigate();
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [name, setName] = useState("");
+
+    function sendRegistration() {
+        const reqBody = {
+          username: username,
+          password: password,
+          name: name,
+        };
+    
+        fetch("api/user/register", {
+          headers: {
+            "Content-Type": "application/json",
+          },
+          method: "post",
+          body: JSON.stringify(reqBody),
+        })
+          .then((response) => response.json).then(data => setPassword(""), setUsername(""), setName(""));
+      }
+
+  return (
+    <div>
+ <Container className="mt-5 ">
+
+ <Row className="justify-content-center">
+          <Col md="8" lg="6">
+            <Form.Group className="mb-3" controlId="name">
+              <Form.Label className="fs-4">Full Name</Form.Label>
+              <Form.Control
+                type="email"

```

Comment:
This input is for a name, not an email. So you'll want to change the type to "text", as opposed to "email", as this will affect the keyboard that comes up on mobile devices.